cross/arm32_ci_script.sh

```diff
@@ -9,23 +9,23 @@ function usage {
     echo '    corefx source is at ~/cfx'
     echo '$ cd ~/cfx'
     echo '$ ./scripts/arm32_ci_script.sh'
```

Comment:
nit: as probably should update this comment but you can do that after this PR is merged.